{
  "cve_id": "CVE-2024-36401",
  "timestamp": "2025-01-14 16:58:45",
  "phases": {
    "primary": {
      "completed": true,
      "timestamp": "2025-01-10 10:40:11"
    },
    "secondary": {
      "completed": true,
      "timestamp": "2025-01-11 20:55:47"
    },
    "extraction": {
      "completed": true,
      "timestamp": "2025-01-12 23:05:11"
    }
  },
  "urls": [
    {
      "url": "https://github.com/Warxim/CVE-2022-41852?tab=readme-ov-file#workaround-for-cve-2022-41852",
      "status": "success",
      "reason": null,
      "timestamp": "2025-01-14 11:33:06"
    },
    {
      "url": "https://github.com/geoserver/geoserver/security/advisories/GHSA-6jj6-gm7p-fcvv",
      "status": "success",
      "reason": null,
      "timestamp": "2025-01-14 11:33:06"
    },
    {
      "url": "https://github.com/geotools/geotools/pull/4797",
      "status": "success",
      "reason": null,
      "timestamp": "2025-01-14 11:33:06"
    },
    {
      "url": "https://github.com/geotools/geotools/security/advisories/GHSA-w3pj-wh35-fq8w",
      "status": "success",
      "reason": null,
      "timestamp": "2025-01-14 11:33:06"
    },
    {
      "url": "https://osgeo-org.atlassian.net/browse/GEOT-7587",
      "status": "unknown",
      "reason": null,
      "timestamp": "2025-01-14 11:33:06"
    },
    {
      "url": "https://www.vicarius.io/vsociety/posts/geoserver-rce-cve-2024-36401",
      "status": "success",
      "reason": null,
      "timestamp": "2025-01-14 11:33:06"
    }
  ]
}